docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc

```diff
@@ -14,13 +14,56 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.7.1>>
 * <<release-notes-8.7.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.7.1 relnotes
+
+[[release-notes-8.7.1]]
+== {fleet} and {agent} 8.7.1
+
+Review important information about the {fleet} and {agent} 8.7.1 release.
+
+[discrete]
+[[new-features-8.7.1]]
+=== New features
+
+The 8.7.1 release adds the following new and notable features.
+
+{agent}::
+* Add support for feature flags, starting with one to toggle fully qualified domain name (FQDN) reporting in
+  events generated by {agent} components, via the `host.name` field {agent-pull}2218[#2218] {agent-issue}2185[#2185]
+
+[discrete]
+[[enhancements-8.7.1]]
+=== Enhancements
+
+{fleet}::
+* The agent policy `Host name format` selector is now enabled by default {kibana-pull}154563[#154563]
+
+[discrete]
+[[bug-fixes-8.7.1]]
+=== Bug fixes
+
+{fleet}::
+* Fixes an issue where the Advanced options toggle in the policy editor always showing {kibana-pull}154612[#154612]
+* Fixes an issue where the warning icon was unable to display in 8.7 {kibana-pull}154119[#154119]
+* Adds updates to output logic {kibana-pull}153226[#153226]
+
+{agent}::
+* Fix parsing of paths from the `container-paths.yml` file used internally when `elastic-agent container` is run {agent-pull}2340[#2340]
+* Fixes a bug that caused delayed "ack" response following Osquerybeat query completions {agent-pull}2406[#2406] {agent-issue}2410[#2410]
+* Fixes a bug that `logging.level` settings were not being respected, coming either from Fleet UI or a config file {agent-pull}2456[#2456] {agent-issue}2450[#2450]
+* Fixes a bug that caused an empty proxy from the {agent} policy to override the proxy set by `--proxy-url` {agent-pull}2468[#2468] {agent-issue}2304[#2304] {agent-issue}2447[#2447]
```

Comment:
```suggestion
* Fixes a bug that caused an empty proxy from a Fleet managed {agent} policy to override the proxy set by `--proxy-url` {agent-pull}2468[#2468] {agent-issue}2304[#2304] {agent-issue}2447[#2447]
```

This is clearer if we specifically mention that the policy has to be Fleet managed. The bug was caused when we introduced support for setting the proxy to use in Fleet.